### PR TITLE
Setup docker and litestream

### PR DESCRIPTION
**Pull request by alisomay**
## For a fuller review please check the next PR
- https://github.com/rust-at-night/rust-at-night-website/pull/25

Aims to close 
- #7
- #16 

The docker setup is not tested yet, I'll quickly do that tomorrow and also not integrated in CI yet due to 
- #8 

not resolved yet.

On the other hand I want to keep PRs small and I think there is enough material to review here. 
Tomorrow if I find time I'd like to test this locally to make sure before the merge. 

If you'd like to check out what litestream is: https://litestream.io/guides/

This PR will likely receive a few commits more and some values are place holders and guesses.

### CODE REVIEW

**Review by berkantsoytas — APPROVED** [review]
LGTM

File: website-backend/docker/Dockerfile
Comment (te-ori) [inline website-backend/docker/Dockerfile:1]: I am asking to learn, why do we need litestream? What is the benefit to use it in this project?

File: website-backend/docker/env.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Please only source this file, don't execute it directly.
+
+AWS_ACCOUNT_ID="012345678901"
+AWS_REGION="eu-central-1"
+ORG_NAME="rust-at-night"
+IMAGE_NAME="website-backend"
```
Comment (te-ori) [inline website-backend/docker/env.sh:8]: If we want to add new sites this name may cause little confusion. Can we change it as homesite-backend or similar?

File: website-backend/docker/docker_build_ci.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euxo pipefail
+
+SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/$(basename "${BASH_SOURCE[0]}")"
+SCRIPT_DIR="$(dirname "$SCRIPT_PATH")"
+
+source "$SCRIPT_DIR/env.sh"
+
+DOCKER_BUILDKIT=1 docker build -f docker/Dockerfile -t "${AWS_ACCOUNT_ID}".dkr.ecr."${AWS_REGION}".amazonaws.com/"${ORG_NAME}"."${IMAGE_NAME}"."${CURRENT_DATE}":git-"${GITHUB_SHA}"
```
Comment (te-ori) [inline website-backend/docker/docker_build_ci.sh:9]: While there is an `if` in `website-backend/docker/docker_build.sh` to check which platform, it is missing in here. Is it important?

**Review by te-ori — APPROVED** [review]

File: website-backend/docker/Dockerfile
Comment (alisomay) [reply website-backend/docker/Dockerfile:1]: Sure. You may check the issue related to ti. Sorry, the name was not very descriptive so probably you have missed it.

- https://github.com/rust-at-night/rust-at-night-website/issues/9

Also if the question is about the need for a database in the project you may derive the context out of this issue.

- https://github.com/rust-at-night/rust-at-night-website/issues/14

File: website-backend/docker/env.sh (hunk 1 above)
Comment (alisomay) [reply website-backend/docker/env.sh:8]: Hmm.. actually it seems that you're right here. 
I'll be updating it.

File: website-backend/docker/docker_build_ci.sh (hunk 2 above)
Comment (alisomay) [reply website-backend/docker/docker_build_ci.sh:9]: I don't think so. 
This file is for CI and we'll be running on linux platforms with x86 as default. 
If there is not a particular reason for doing so I think we can assume the arch and platform by default.

**Comment by dvtkrlbs** [comment]
LGTM

**Review by dvtkrlbs — APPROVED** [review]

**Pull request merged**